drivers/SmartThings/zigbee-smoke-detector/config.yml

```diff
@@ -1,5 +1,5 @@
-name: 'Zigbee Smoke Detector'
-packageKey: 'zigbee-smoke-detector'
+name: 'Zigbee Smoke Detector by Gabriel'
+packageKey: 'zigbee-smoke-detector by Gabriel'
```

Comment:
@GAFfrient Since your contributing code into the SmartThings owned Edge Drivers, this can be removed